herbertUG/monsterGame
# implement functionality and design to the game.

**What does the PR do?**
This PR adds functionality to the game.

**Description of task?**
Implement functionality where a user is able to interact with the game and return scores.

**How should this be manually tested?**
- Clone the repository.
- Then check out to the `develop` branch after which run the application via the browser.
